lib/order.rb

```diff
@@ -0,0 +1,55 @@
+require_relative 'menu'
+
+class Order
+  attr_reader :menu, :cost, :items
+  DELIVERY_TIME = 1800
+
+  def initialize(menu = Menu.new)
+    @menu = menu.list
+    @items = {}
+    cost = 0
+  end
+
+  def update(items)
```

Comment:
Perhaps change the name of this method to `add_to_order` etc. At the moment it seems slightly vague as unsure if it is ACTUALLY updating something or adding to something?